pages/endpoint/workflow.mdx

```diff
@@ -0,0 +1,143 @@
+import { OptionTable } from '@components/table'
+
+# Workflow
+
+The Workflow class provides methods for executing and managing workflow tasks, such as image upscaling.
+
+## Execute Workflow and Wait for Result
+
+Executes a workflow task and waits for the result.
+
+### Input
+
+```ts {8-14} copy
+import Heurist from 'heurist'
+
+const heurist = new Heurist({
+  apiKey: process.env['HEURIST_API_KEY'], // This is the default and can be omitted
+})
+
+async function main() {
+  const upscalerTask = new UpscalerTask({
+    consumer_id: 'example-id',
+    workflow_id: '1',
```

Comment:
Is there 1:1 mapping between workflow ID and Task? Like upscaling workflow is ID1, animation workflow is ID2, and so on, defined in the server/miner side? If so, we can make the workflow ID a static variable of the Task class

Reply:
No such mapping now, the original idea is miner side will distinguish different task type and send it to correct docker service. User end actually no need to send workflow_id, just use task type.

Reply:
I thought the workflow_id can distinguish different versions of, say, UpscalerTask (like 1 being simple upscaler, 2 being more complex upscaler to 4k or something)? Is there such a use case? If not then we just need task type

Reply:
wil remove workflow_id as discussed on TG